deprecated/content/en/docs/user-journeys/users/application-developer/intermediate.md

```diff
@@ -0,0 +1,166 @@
+---
+reviewers:
+- chenopis
+layout: docsportal
+css: /css/style_user_journeys.css
```

Comment:
Hi @aidanranney .
Is there a reason to keep the content in "deprecated"?
If not, I would remove the directory and files.

Reply:
@kbhawkey Can we follow this up in an iterative PR?

Reply:
👍